README.md

```diff
@@ -33,10 +33,8 @@ nile init
 ✅ Dependencies successfully installed
 🗄  Creating project directory tree
 ⛵️ Nile project ready! Try running:
-
-nile compile
```

Comment:
I think this was a runaway line. Let me know if it was there for a reason.

Reply:
it was meant to work as an example command you can run with nile (notice the colon in the line above)